Removed redundant cast

matbech opened the pull request:
The cast is redundant.

Trivial change.

matbech committed:
Removed redundant cast

The cast is redundant.

Xerkus commented on src/Http/Segment.php:
```diff
@@ -446,7 +446,7 @@ public function getAssembledParams()
     protected function encode(string $value)
     {
-        $key = (string) $value;
+        $key = $value;
```
@matbech separate `$key` and `$value` variables are redundant as well now.

Xerkus reviewed (changes requested):
This is a patch level change, it should be done against 3.4.x branch for the next patch level release. Please rebase.

Also, for your commit to be included we require commits to be signed off for [Developer Certificate of Origin](https://developercertificate.org). Please see DCO check.

matbech commented:
Please go ahead with the required changes. The whole process (requirements) of doing a trivial change is ridiculous.

weierophinney committed:
refactor: no need to rename variable

$value can be used where $key was used previously, as we can assume it is a string.

Signed-off-by: Matthew Weier O'Phinney <matthew@weierophinney.net>

matbech commented:
Thank you
